src/index.js

```diff
@@ -3,26 +3,32 @@
 // eslint-disable-next-line import/extensions, import/no-extraneous-dependencies
 import { CompositeDisposable } from 'atom';
 import path from 'path';
-import escapeHtml from 'escape-html';
 import pluralize from 'pluralize';
 import * as helpers from 'atom-linter';
+import { get } from 'request-promise';
 
 const DEFAULT_ARGS = [
   '--cache', 'false',
   '--force-exclusion',
   '--format', 'json',
   '--display-style-guide',
 ];
+const DOCUMENTATION_LIFETIME = 86400 * 1000; // 1 day TODO: Configurable?
 
-const formatMessage = ({ message, cop_name: copName, url }) => {
-  const formattedMessage = escapeHtml(message || 'Unknown Error');
-  let formattedCopName = '';
-  if (copName && url) {
-    formattedCopName = ` (<a href="${escapeHtml(url)}">${escapeHtml(copName)}</a>)`;
-  } else if (copName) {
-    formattedCopName = ` (${escapeHtml(copName)})`;
+let docsRuleCache = {};
```

Comment:
Might make things cleaner to implement this in a [`Map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map).